staging/src/k8s.io/apiserver/pkg/storage/tests/cacher_test.go

```diff
@@ -323,7 +323,7 @@ func TestInfiniteList(t *testing.T) {
 
 	result := &example.PodList{}
 	err = cacher.List(context.TODO(), "pods/ns", listRV, storage.Everything, result)
-	if !errors.IsTimeout(err) {
+	if !storage.IsTooLargeResourceVersion(err) {
```

Comment:
the test name also needs to be fixed. i assume perviously it will retry forever.

Reply:
hmm... ok after reading watch_cache.go, i understand we still do retry. but i guess changing the test name to make this less confusing is better anyway.

Reply:
I agree. I've renamed the test and added a comment to clarify further.